server/backend/api/app/routers/router_user_terms_conditions.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from http import HTTPStatus
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from api.app import database
+from api.app.schemas import Requester, FamUserTermsConditionsGet
+from api.app.crud import crud_user_terms_conditions
+from api.app.routers.router_guards import get_current_requester
+
+LOGGER = logging.getLogger(__name__)
+router = APIRouter()
+
+
+@router.get(
+    "/check",
```

Comment:
I'm not sure how to name the path, we can rename it

Reply:
This is an interesting one. 
I will use "**POST**", that's what most commonly seen for this kind of not reqular REST resource requests.

But for naming, I might opt for "**validate**" instead.
However, it might be good to follow some [convention](https://restfulapi.net/resource-naming/) (maybe this case has exception). At the end of the doc it mentioned it can be (POST having action=validate in the body):
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/d79a7cdb-67fb-4e97-8a74-f0941876413a)

Or google has its convention using (":") for "**[custom methods](https://cloud.google.com/apis/design/custom_methods)**" which it would look like 
"**POST /user_terms_conditions/user:validate**"
But I am not sure how FastAPI will have no issue with that or not, or simply it is just another endpoint handler as (/user_terms_conditions/user:validate --> `@router.post("user:validate")`)

Perhaps, most commonly seen is using (requestBody:  action=validate)

Reply:
so maybe POST with `/user_terms_conditions/{version_id}/verify-user` ?

Reply:
<img width="916" alt="Screen Shot 2024-06-12 at 1 58 07 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/339a0bb0-5b76-42f3-bd1f-12e3476c6326">

Reply:
maybe this way? 

<img width="1070" alt="Screen Shot 2024-06-12 at 2 21 57 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/9a9b2c2d-458e-4480-b1ef-17245588c107">

Reply:
This is good I feel.